rest_test.go

```diff
@@ -95,7 +95,7 @@ func TestRest(t *testing.T) {
 		Headers:     Headers,
 		QueryParams: queryParams,
 	}
-	response, e := API(request)
```

Comment:
Please add these tests back in, in addition to the new ones you created for `Send`. Thanks!

Reply:
I add function `testingAPI`, and send to this function like parameter functions `Send` and `API` because code the same.